### ISSUE TITLE
parse does not work with ${configDir} variable

### ISSUE THREAD

**Issue by Zamiell** [dominikg/tsconfck#188]
steps to reproduce:

```
git clone git@github.com:IsaacScript/isaacscript.git
cd isaacscript
git checkout 939662fcb5d5efc37a80b2cf8e5f7f4e23986c9e
yarn
cd packages/eslint-plugin-isaacscript
npm run build
# The build will succeed.
# Now, we change "tsconfck.parseNative" to "tsconfck.parse".
sed --in-place 's/tsconfck\.parseNative/tsconfck.parse/g' ../isaacscript-common-node/src/functions/scriptHelpers.ts
npm run build
# The build fails.
```

**Comment by dominikg** [comment]
please provide a link to a minimal reproduction, not a huge monorepo.

**Comment by Zamiell** [comment]
Sorry, I don't currently have the cycles to provide a more minimal reproduction. For now, I will use `parseNative`, and I guess we can leave this issue open to see if anyone else has the same problem.

**Comment by Zamiell** [comment]
I think I narrowed down the bug. It seems that `tsconfck` does not parse npm packages correctly, as demonstrated by the following reprex: https://github.com/Zamiell/test/tree/main

If you copy paste [the remote tsconfig](https://github.com/IsaacScript/isaacscript/blob/main/packages/isaacscript-tsconfig/tsconfig.base.json) and put it in the repo root, then everything works fine. Thus, I guess the logic for resolving npm packages isn't working properly?

**Comment by dominikg** [comment]
please check #189 and lmk if that works for you. I did add a testcase based on your config so it really should ;) Thanks for the reproduction

**Comment by Zamiell** [comment]
thanks, that does seem to fix the problem!

**Comment by dominikg** [comment]
fix released in 3.1.2
